pkg/controller/util/node/controller_utils.go

```diff
@@ -80,6 +80,11 @@ func DeletePods(kubeClient clientset.Interface, pods []*v1.Pod, recorder record.
 		klog.V(2).Infof("Starting deletion of pod %v/%v", pod.Namespace, pod.Name)
 		recorder.Eventf(pod, v1.EventTypeNormal, "NodeControllerEviction", "Marking for deletion Pod %s from Node %s", pod.Name, nodeName)
 		if err := kubeClient.CoreV1().Pods(pod.Namespace).Delete(pod.Name, nil); err != nil {
+			if apierrors.IsNotFound(err) {
+				// Not founding pods is something that can happen in a cluster.
+				// This handling is crucial for correctness.
```

Comment:
This comment isn't particularly useful for future reader. Why it is crucial?

// NotFound error means that pod was already deleted. There is nothing left to do with this pod.

Reply:
I meant replace your whole comment with that, not just the second line.]

Reply:
Done.